systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -43,11 +43,18 @@ public static void waitUntilClientReceivedMessagesTls(KafkaClientOperations kafk
     }
 
     public static void waitTillContinuousClientsFinish(String producerName, String consumerName, String namespace, int messageCount) {
-        long timeout = (long) messageCount * 1000;
+        long timeout = (long) messageCount * 3000;
         LOGGER.info("Waiting till producer {} and consumer {} finish for the following {} ms", producerName, consumerName, timeout);
         TestUtils.waitFor("continuous clients finished", Constants.GLOBAL_POLL_INTERVAL, timeout,
             () -> kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(producerName).get().getStatus().getSucceeded().equals(1) &&
                     kubeClient().getClient().batch().jobs().inNamespace(namespace).withName(consumerName).get().getStatus().getSucceeded().equals(1));
     }
+
+    public static void waitTillProducerOrConsumerFinish(String jobName, String namespace, int messageCount) {
```

Comment:
It's quite strange method name. Do you wait for producer or consumer in the test? I think you should name it like waitTillClientFinish instead of this.